gems/smithy/lib/smithy/templates/client/types.erb

```diff
@@ -8,6 +8,7 @@ module <%= namespace %>
 <% types.each do |type| -%>
 
     <%= type.documentation %>
+<% if type.type == 'structure' -%>
     <%= type.name %> = Struct.new(
```

Comment:
Did we add camelize to this one?

Reply:
the type? No - thats the raw smithy type.

If you mean the name on the next line - yes.